lib/legacy/class.html

```diff
@@ -63,6 +63,7 @@
         behaviors = superBehaviors.concat(behaviors);
       }
       klass.behaviors = behaviors;
+      klass.prototype.behaviors = behaviors;
```

Comment:
not sure if this should belong to the mixin or the legacy element, same for `properties`.
Should I just have two getters in the LegacyElementMixin?